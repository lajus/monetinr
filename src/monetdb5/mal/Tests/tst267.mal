# Grouped sums
a:= bat.new(:oid,:int);
b:= bat.new(:oid,:int);
bat.append(a,1);
bat.append(a,2);
bat.append(b,5);
bat.append(b,6);
bat.append(b,7);
bat.append(b,8);
io.print(a);
io.print(b);

g:= array.product(a,b);
io.print(g);
