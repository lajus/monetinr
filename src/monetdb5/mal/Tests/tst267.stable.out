stdout of test 'tst267` in directory 'monetdb5/mal` itself:


# 20:55:59 >  
# 20:55:59 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=40036 --set sql_port=52173 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst267.mal </dev/null
# 20:55:59 >  

# Monet Database Server V4.7.0
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# config:/ufs/mk/monet5/src/mal/Tests/All.conf
# dbfarm:/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
## Grouped sums
#a:= new(:void,:int);
#setSequenceBase(a,0:oid);
#b:= new(:void,:int);
#setSequenceBase(b,0:oid);
#insert(a,1);
#insert(a,2);
#insert(b,3);
#insert(b,4);
#insert(b,5);
#print(a);
#print(b);
#
#g:= array.*(a,b);
#print(g);
#
function user.main():void;
# Grouped sums 
    a := bat.new(:oid,:int);
    b := bat.new(:oid,:int);
    bat.append(a,1);
    bat.append(a,2);
    bat.append(b,5);
    bat.append(b,6);
    bat.append(b,7);
    bat.append(b,8);
    io.print(a);
    io.print(b);
    g := array.product(a,b);
    io.print(g);
end main;
#-----------------#
# h	tmp_13	  # name
# void	int	  # type
#-----------------#
[ 0@0,	  1	  ]
[ 1@0,	  2	  ]
#-----------------#
# h	tmp_14	  # name
# void	int	  # type
#-----------------#
[ 0@0,	  5	  ]
[ 1@0,	  6	  ]
[ 2@0,	  7	  ]
[ 3@0,	  8	  ]
#-----------------#
# h	tmp_15	  # name
# void	int	  # type
#-----------------#
[ 1,	  5	  ]
[ 2,	  6	  ]
[ 1,	  7	  ]
[ 2,	  8	  ]

# 20:55:59 >  
# 20:55:59 >  Done.
# 20:55:59 >  

